src/getConfig.js

```diff
@@ -61,12 +61,10 @@ function unknownValidationReporter(config, example, option, options) {
       format(config[option], { inlineCharacterLimit: Number.POSITIVE_INFINITY })
     )} was found in the config root.
   
-  You probably trying to mix simple and advanced config formats. 
-  
-  Adding 
+  You are probably trying to mix simple and advanced config formats. Adding 
   
   ${chalk.bold(`"linters": {
-    "${option}": ${format(config[option], { inlineCharacterLimit: Number.POSITIVE_INFINITY }, '  ')}
+    "${option}": ${JSON.stringify(config[option], null, '\t    ')}
```

Comment:
This results in inconsistent indentation. Massaged snapshot equivalent:
```
  "linters": {
    "*.js": [
	    "eslint --fix",
	    "git add"
]
```

Reply:
Yeah, I've noticed that. Do you know how to fix that?

Reply:
Don't add the indentation while formatting using `JSON.stringify`. This will print the commands on a single line but that's okay I think.

Reply:
Hmm, okay, I guess that can be improved later on